spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/token/RemoteTokenServices.java

```diff
@@ -61,6 +61,14 @@
 
 	public RemoteTokenServices() {
 		restTemplate = new RestTemplate();
+		((RestTemplate)restTemplate).getInterceptors().add(new ClientHttpRequestInterceptor() {
+			@Override
+			// Only accept json
+			public ClientHttpResponse intercept(HttpRequest request, byte[] body, ClientHttpRequestExecution execution) throws IOException {
+				request.getHeaders().setAccept(Arrays.asList(MediaType.APPLICATION_JSON));
```

Comment:
This would be very restrictive. The Authorization Server may return another format other than JSON. If you require it to be JSON then you can configure the `RestTemplate` and supply it via `RemoteTokenServices.setRestTemplate()`.